File: task2.js
// Maximum and minimum of an array

function getMinMax(arr) {

    let minMax = {
        min: arr[0],
        max: arr[0]
    }
    arr.map((edx) => {
        if (minMax.min > edx) {
            minMax.min = edx
        }
        else if (minMax.max < edx) {
            minMax.max = edx
        }
    })

    return minMax


}


let arr = [-1, -223, 23, 53, 12, 4, 9]

let minMax = getMinMax(arr);

console.log("Minimum Value is " + minMax.min)
console.log("Maximum Value is " + minMax.max)

File: task2.py

# Maximum and minimum of an array

def findMinMax(arr):
    print(min(arr))
    print(max(arr))
    print("--------------------------------")

    minValue = arr[0]
    maxValue = arr[0]

    for i in range(len(arr)):
        if (minValue > arr[i]):
            minValue = arr[i]
        elif (maxValue < arr[i]):
            maxValue = arr[i]

    print(minValue)
    print(maxValue)


arr = [1, 4, 534, 234, -1]

findMinMax(arr)
